examples/lv_example_chart_1.py

```diff
@@ -0,0 +1,54 @@
+"""
+slightly modified lv_example_chart_1.py
```

Comment:
How is this different from lv_example_chart_1.py?   
If the only difference is `import portrait_128x160 as driver` and `chart.set_size(driver.disp.width, driver.disp.height)`, then I'm not sure I see the extra benefit over lv_example_chart_1.py.